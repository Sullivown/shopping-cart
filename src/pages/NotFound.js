import React, { useEffect } from 'react';
import updateDocumentTitle from '../helpers/updateDocumentTitle';

function NotFound(props) {
	useEffect(() => {
		updateDocumentTitle(props.title);
	});

	return (
		<div className='not-found'>
			<h2>404 - page not found!</h2>
		</div>
	);
}

export default NotFound;
