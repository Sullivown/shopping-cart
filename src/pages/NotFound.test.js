import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import NotFound from './NotFound';

describe('Test NotFound component', () => {
	it('renders', () => {
		const notFoundTestRenderer = TestRenderer.create(<NotFound />).toJSON();
		expect(notFoundTestRenderer).toMatchSnapshot();
	});
});
