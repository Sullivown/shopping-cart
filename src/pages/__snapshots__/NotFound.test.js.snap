// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NotFound component renders 1`] = `
<div
  className="page-content not-found"
>
  <img
    alt="Sad cloud"
    src="sad-cloud.png"
  />
  <h2>
    404 - page not found!
  </h2>
</div>
`;
